io.github.zzzgydi.ClashVerge.yml

```diff
@@ -0,0 +1,82 @@
+app-id: io.github.zzzgydi.ClashVerge
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: clash-verge
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+
+modules:
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.38.6.tar.xz
+        sha256: 1c614c9589389db1a79ea9ba4293bbe8ac3ab0a2234cac700935fae0724ad48b
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: <a href="webkitgtk\-(2\.38\.\d+)\.tar\.xz">
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPORT=GTK
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DUSE_SOUP2=ON
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+      - name: bubblewrap
+        sources:
+          - type: archive
+            url: https://github.com/containers/bubblewrap/releases/download/v0.8.0/bubblewrap-0.8.0.tar.xz
+            sha256: 957ad1149db9033db88e988b12bcebe349a445e1efc8a9b59ad2939a113d333a
+            x-checker-data:
+              type: json
+              url: https://api.github.com/repos/containers/bubblewrap/releases/latest
+              tag-query: .tag_name
+              timestamp-query: .published_at
+              version-query: $tag | sub("^v"; "")
+              url-query: '"https://github.com/containers/bubblewrap/releases/download/\($tag)/bubblewrap-\($version).tar.xz"'
+        buildsystem: meson
+      - name: xdg-dbus-proxy
+        sources:
+          - type: archive
+            url: https://github.com/flatpak/xdg-dbus-proxy/releases/download/0.1.4/xdg-dbus-proxy-0.1.4.tar.xz
+            sha256: 1ec0eab53d1e49966d722352bcfd51ac402dce5190baedc749a8541e761670ab
+            x-checker-data:
+              type: json
+              url: https://api.github.com/repos/flatpak/xdg-dbus-proxy/releases/latest
+              tag-query: .tag_name
+              timestamp-query: .published_at
+              version-query: $tag
+              url-query: '"https://github.com/flatpak/xdg-dbus-proxy/releases/download/\($tag)/xdg-dbus-proxy-\($version).tar.xz"'
+        buildsystem: meson
+
+  - name: ClashVerge
+    sources:
+      - type: file
+        url: https://github.com/zzzgydi/clash-verge/releases/download/v1.3.3/clash-verge_1.3.3_amd64.deb
+        sha256: f55b070b41ea57709abea070f6e2becc2d671c890af03af2df384ad6bafe5415
+      
+      - type: file
+        path: io.github.zzzgydi.ClashVerge.png
```

Comment:
why adding the icon here when it there: https://github.com/zzzgydi/clash-verge/blob/main/src-tauri/icons/128x128.png